server/src/dev-data/seeds/csvGenerator.js

```diff
@@ -14,13 +14,13 @@ const { generateMockCSVReward } = require('./generator');
 // our stream that will be passed as the first argument for our generator function as 'writer'
 const writeProjects = fs.createWriteStream('projects.csv');
 // write our CSV headers
-writeProjects.write('rewardID,title,pledgeAmount,description,deliveryMonth,deliveryYear,shippingType,rewardQuantity,timeLimit,randomId,rewardItems\n','utf8');
+writeProjects.write('rewardID;title;pledgeAmount;description;deliveryMonth;deliveryYear;shippingType;rewardQuantity;timeLimit;randomId;rewardItems\n','utf8');
```

Comment:
interesting choice to use ; as a delimiter. If the change was made to upload things into Postgres, there are a few alternatives to modifying your generation to be able to handle commas.